### PR TITLE
Running ding dong bot on gitpod

### PR DESCRIPTION
**Pull request by shwetalsoni**
Added tutorial for **Running on Gitpod**

Below is the structure that I have followed:

* Create an account on Gitpod if a new user
* Creating workspace
* Scanning the QR code
* Connecting the bot with specified puppet
* Testing the bot

Screenshot from 2021-06-05 15-22-27
Screenshot from 2021-06-05 15-22-52
Screenshot from 2021-06-05 15-23-07
Screenshot from 2021-06-05 15-23-20

### CODE REVIEW

**Comment by shwetalsoni** [comment]
Tag: #891

File: docusaurus/docs/getting-started/running-on-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Running on Gitpod
+---
+
+If you are total begginer to wechaty this tutorial will guide you how to run the [Ding Dong bot](https://github.com/wechaty/wechaty-getting-started/blob/master/examples/ding-dong-bot.ts) from the [Wechaty Getting Started](https://github.com/wechaty/wechaty-getting-started) GitHub repository directly on the browser using **Gitpod**.
```
Comment (proudofsimin) [inline docusaurus/docs/getting-started/running-on-gitpod.md]: If you are total ~begginer~ [beginner] to ~w~ [W] echaty [,]  this tutorial will guide you...

File: docusaurus/docs/getting-started/running-on-gitpod.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Gitpod is an online and open source platform for automated and ready-to-code development environments.
+
+Follow the steps below to get started.
```
Comment (proudofsimin) [inline docusaurus/docs/getting-started/running-on-gitpod.md]: Add a colon after *to get started*.

File: docusaurus/docs/getting-started/running-on-gitpod.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  [![gitpod-link][gitpod_img]][gitpod_link]
+
+  [gitpod_img]: https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod
+  [gitpod_link]: https://gitpod.io/#https://github.com/wechaty/wechaty-getting-started
+
+1. If you have never used gitpod before, you will be required to login using your gitHub account.
+
+  ![sign in](../../static/img/docs/getting-started/quick-start/gitpod/gitpod-signin.png)
+
+1. After logging in with your github account you have to create a workspace by clicking on **New Workspace** button.
```
Comment (proudofsimin) [inline docusaurus/docs/getting-started/running-on-gitpod.md]: After logging in with your github account [,] you have to create a workspace by clicking on **New Workspace** button.

**Review by proudofsimin — APPROVED** [review]
Great but please correct some typos.

**Comment by shwetalsoni** [comment]
@proudofsimin I have made the suggested changes.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Perfect PR : clean commit history, properly based, relevant screenshots, PR good in scope.

File: docusaurus/docs/getting-started/running-on-gitpod.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+---
+title: Running on Gitpod
+---
+
+If you are total beginner to wechaty this tutorial will guide you how to run the [Ding Dong bot](https://github.com/wechaty/wechaty-getting-started/blob/master/examples/ding-dong-bot.ts) from the [Wechaty Getting Started](https://github.com/wechaty/wechaty-getting-started) GitHub repository directly on the browser using **Gitpod**.
```
Comment (lijiarui) [inline docusaurus/docs/getting-started/running-on-gitpod.md]: How about we add Gitpod link here, change to：

[Gitpod](https://gitpod.io/#https://github.com/wechaty/wechaty-getting-started)

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request, thanks!

**Comment by shwetalsoni** [comment]
@lijiarui I have made the requested change.

**Review by lijiarui — APPROVED** [review]
Good job!

**Review by huan — APPROVED** [review]

**Comment by lijiarui** [comment]
@shwetalsoni Please resolve this conflict and then I will merge this PR.

**Pull request merged**